python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -1277,6 +1277,75 @@ def _makeArgumentParser(cls):
         return parser
 
 
+class ImageDifferenceFromTemplateConnections(ImageDifferenceTaskConnections,
+                                             defaultTemplates={"coaddName": "goodSeeing"}
+                                             ):
+    inputTemplate = pipeBase.connectionTypes.Input(
+        doc=("Warped template produced by GetMultiTractCoaddTemplate. ",
+             "Replaces need for coaddExposures"),
```

Comment:
This comment is slightly misleading. You still need to have coaddExposures in order to build the template, you just don't have to supply them to this Task since that work is already done.